web-wallet/src/lib/components/Transactions/Transactions.svelte

```diff
@@ -68,59 +74,52 @@
       />
     {/if}
   </header>
-  <Suspense
-    className="transactions-list__container"
-    errorMessage="Error getting transactions"
-    errorVariant="details"
-    waitFor={items}
-  >
-    <svelte:fragment slot="success-content" let:result={transactions}>
-      {#if transactions.length}
-        {#each getOrderedTransactions(transactions) as transaction (transaction.id)}
-          <dl class="transactions-list">
-            <dt class="transactions-list__term">Hash</dt>
-            <dd class="transactions-list__datum transactions-list__datum--hash">
-              <samp>
-                <AppAnchor
-                  href="https://explorer.dusk.network/transactions/transaction?id={transaction.id}"
-                  rel="noopener noreferrer"
-                  target="_blank"
-                >
-                  {middleEllipsis(
-                    transaction.id,
-                    calculateAdaptiveCharCount(screenWidth, 320, 640, 5, 20)
-                  )}
-                </AppAnchor>
-              </samp>
-            </dd>
-            {#if transaction.tx_type}
-              <dt class="transactions-list__term">Type</dt>
+  {#if !isSyncing && !error}
```

Comment:
Not saying it's wrong, just wanted to point out that, if there is a sync error, we will still display the message "Data will load after syncing has finished.", even if the sync has finished with an error.

I think it's fine, the user will see the sync has failed anyways from the sync status.

Reply:
I'm a bit confused about what you are saying here. Not sure I'm following.

Reply:
> Not saying it's wrong, just wanted to point out that, if there is a sync error, we will still display the message "Data will load after syncing has finished.", even if the sync has finished with an error.
> 
> I think it's fine, the user will see the sync has failed anyways from the sync status.

One way to approach this would be by having a custom content for the `Suspense`'s `pending-content` slot, and make the check (only the `isSyncing` one, ignoring the error) inside that, instead of conditionally render the whoel `Suspense`.

Reply:
@laremas I think this should be addressed in the interaction design.